The 4465 release broke building stable releases on Linux

### My mods and Everest install are up to date

Yes

### I have recreated the bug with only Everest OR a minimum number of mods enabled

Yes

### Describe the bug

I've been building Everest releases from source code using the stable releases on Github. With 4465, this stopped working.

### Steps to reproduce

E.g.

1. `wget "https://github.com/EverestAPI/Everest/archive/refs/tags/stable-1.4465.0.tar.gz"`
2. `tar xf stable-1.4465.0.tar.gz`
3. `cd Everest-stable-1.4465.0`
4. `dotnet build --nologo --verbosity quiet "/p:Configuration=Release"`

I see a series of errors that look something like this:

/home/adam/downloads/everest/Everest-stable-1.4465.0/NETCoreifier/Coreifier.cs(3,7): error CS0246: The type or namespace name 'MonoMod' could not be found (are you missing a using directive or an assembly reference?) [/home/adam/downloads/everest/Everest-stable-1.4465.0/NETCoreifier/NETCoreifier.csproj]

All the errors I see (91 of them) are in the NetCoreifier project. At first I thought the issue had to do with missing submodules, but (a) this isn't an issue for previous stable releases, and (b) manually cloning monomod into the `external` directory doesn't fix the issue.

Incidentally, building the git version now fails as well unless I install the 6.0 version of the .NET framework. This isn't an issue for building the 4449 release - I have a working Celeste + Everest install on my system that I built with exclusively .NET 8.0 tooling.

### Expected behavior

The build succeeds. In fact, if you download the previous stable release (4449), the above series of commands works just fine.

### Operating System

Arch Linux

### Everest Version

4465

### Mods required to reproduce

_No response_

### Additional context

_No response_

--- a/README.md
+++ b/README.md
@@ -21,8 +21,9 @@ For information about mod development, check out the [Everest Wiki](https://gith
 - ***If you just want to install Everest, go to the [Everest website](https://everestapi.github.io/).***
 - If you've **previously installed Everest and updated Celeste** or switched betas / branches, delete the `orig` directory where Celeste is installed.
     - macOS: Right-click and browse the Celeste app in Finder, then navigate to `Contents`, then `MacOS`.
-- Clone the Everest repo **+ submodules**, either in your IDE, via the CLI, or by downloading the .zip from GitHub.
+- Clone the Everest repo **+ submodules**, either in your IDE or via the CLI.
 - Restore Nuget packages either via your IDE or the command line.
+- Everest requires both 6.0 and 7.0+ versions of the .NET Core SDK for the build process.
 
 ### Windows
 - Open the .sln in the repo with Visual Studio

--- a/README.md
+++ b/README.md
@@ -23,7 +23,7 @@ For information about mod development, check out the [Everest Wiki](https://gith
     - macOS: Right-click and browse the Celeste app in Finder, then navigate to `Contents`, then `MacOS`.
 - Clone the Everest repo **+ submodules**, either in your IDE or via the CLI.
 - Restore Nuget packages either via your IDE or the command line.
-- Everest requires version 7.0.200 or higher of .NET Core SDK, as well as the .NET 6.0 Core runtime, for the build process.
+- Everest requires version 7.0.200 or higher of .NET SDK, as well as the .NET 6.0 runtime, for the build process.
 
 ### Windows
 - Open the .sln in the repo with Visual Studio

--- a/README.md
+++ b/README.md
@@ -23,7 +23,7 @@ For information about mod development, check out the [Everest Wiki](https://gith
     - macOS: Right-click and browse the Celeste app in Finder, then navigate to `Contents`, then `MacOS`.
 - Clone the Everest repo **+ submodules**, either in your IDE or via the CLI.
 - Restore Nuget packages either via your IDE or the command line.
-- Everest requires both 6.0 and 7.0+ versions of the .NET Core SDK for the build process.
+- Everest requires version 7.0.200 or higher of .NET Core SDK, as well as the .NET 6.0 Core runtime, for the build process.
 
 ### Windows
 - Open the .sln in the repo with Visual Studio

